src/Farfetch.LoadShedding/Limiters/AdaptativeConcurrencyLimiter.cs

```diff
@@ -48,39 +48,37 @@ internal class AdaptativeConcurrencyLimiter : IAdaptativeConcurrencyLimiter, IDi
         /// It is responsible for managing the adaptative semaphore for the current request.
         /// As soon as the request is executed, the data is collected to determine the performance degradation.
         /// </summary>
-        public Task ExecuteAsync(Func<Task> function, CancellationToken cancellationToken = default)
+        public Task ExecuteAsync(Func<Task> function, string method = null, CancellationToken cancellationToken = default)
         {
-            return this.ExecuteAsync(0, function, cancellationToken);
+            return this.ExecuteAsync(Priority.Normal, function, method, cancellationToken);
```

Comment:
**question for maintainers**: Regarding the first commit, should the default priority be Normal or should we keep it as a Critical(0) priority?

Reply:
The default priority should be normal.